docs/Porting_Guide.md

```diff
@@ -35,8 +35,9 @@ Understanding the TizenRT code structure and adhering to the structure helps in
 1. [How to use SmartFS](HowToUseSmartFS.md)
 
 ## Network
-1. [How to use LWIP](HowToUseLWIP.md)
-2. [How to port TizenRT on new WiFi chipset](HowToPortTizenRTOnWiFiChipset.md)
+1. [How to port TizenRT on new WiFi chipset](HowToPortTizenRTOnWiFiChipset.md)
+2. [How to use LWIP](HowToUseLWIP.md)
+2. [How to test network stack](HowToTestNetworkPerformance.md)
```

Comment:
There is the `Confirmation of Porting` tab. How do you think keeping it as separated tab or merged into each module as you did?

Reply:
Separation is a great idea. I think it is more logically natural. Thanks for the comment.